kubetest/extract_test.go

```diff
@@ -83,3 +83,58 @@ func TestGetKube(t *testing.T) {
 		}
 	}
 }
+
+func TestExtractStrategies(t *testing.T) {
+	cases := []struct {
+		option        string
+		expectURL     string
```

Comment:
It is always "" for the existing bazel tests. I was trying to be slightly forward-thinking for when someone wants to add a new unit test for a different extraction strategy, since this is written generically enough to just keep adding test cases to the list, but I can tear this out until it's actually needed if it's confusing.